README.md

```diff
@@ -36,6 +37,16 @@ got it setup. To test this works try cloning a small private repository, e.g.
 $ git clone ssh://git@github.com/chime-experiment/mkchimeenv
```

Comment:
This repo is no longer private, so it would probably be good to suggest something else in here.

Reply:
Oops, good catch. I've changed this to `chimedb_config`, which is small and private